ppr-ui/src/components/parties/party/PartyAutocomplete.vue

```diff
@@ -227,6 +248,11 @@ export default defineComponent({
   color: $gray7 !important;
 }
 
+.wasSelected {
```

Comment:
CSS class names usually have dash/hyphens.

Reply:
updated to `was-selected`